paasta_tools/chronos_tools.py

```diff
@@ -392,7 +392,10 @@ def format_chronos_job_dict(self, docker_url, docker_volumes, dockerfile_locatio
                 'image': docker_url,
                 'network': net,
                 'type': 'DOCKER',
-                'volumes': docker_volumes
+                'volumes': docker_volumes,
+                'parameters': {
+                    'memory-swap': "%sm" % self.get_mem(),
```

Comment:
The docker man page says this "always" must be larger than -m, can you confirm it actually works if this is the same?


Also can you update the docs about "how paasta runs docker images to say that paasta doesn't allow swap?